### PR TITLE
Fix/pdirefresh

### PR DESCRIPTION
**Pull request by luizcarlosdk**

### CODE REVIEW

**Comment by luizcarlosdk** [comment]
There is 2 things to change, the "keywords = row[13]?.split(";")?.toSet()" is giving out-of-bound due the fact that the PDI List length is 12, but we do have this information on auspin drive.

fixed the first issue, we need to show this keywords in frontend.

**Comment by luizcarlosdk** [comment]
error print:

image

File: frontend/modules/Catalog/pages/inovacao.vue [hunk 1]
```diff
@@ -34,6 +25,7 @@
         <p class="body-2 font-italic">{{ item.category }}</p>
         <p class="body-2">{{ item.unity }}</p>
         <p class="body-2">{{ item.campus }}</p>
+        <p class="body-2">{{ item.keywords.join(';') }}</p>
```
Comment (jooaodanieel) [inline frontend/modules/Catalog/pages/inovacao.vue:28]: Vale a pena dar join por `"; "`(com espaço depois do ponto-e-vírgula) ao invés de `";"` (sem espaço)

**Review by jooaodanieel — CHANGES_REQUESTED** [review]
Ponto mais importante: testar de fato o comportamento de limpar o BD no RefreshPDI.

**Review by jooaodanieel — APPROVED** [review]
The major issue is, actually, something small (which is good, means the PR is of quality): the same serialization code repeats 3 times in different places of the application. I left a food-for-thought in the first occurrence.
The other issue is related to the interval considered for deleting data (of 5 seconds).

Think about those things. If any other change is felt as necessary, do it. Otherwise, I'm leaving the PR review as approved, so you can merge it without any further changes.

**Pull request merged**